### PR TITLE
call PRIVATE_KEY_UNLOCK() in wolfengine_bind for FIPS.

### PR DESCRIPTION
**Pull request by anhu**

### CODE REVIEW

**Comment by anhu** [comment]
This depends on https://github.com/wolfSSL/wolfEngine/pull/208

File: src/we_internal.c [hunk 1]
```diff
@@ -1375,6 +1375,7 @@ int wolfengine_bind(ENGINE *e, const char *id)
 
 #ifdef HAVE_FIPS
     wolfCrypt_SetCb_fips(we_fips_cb);
+    PRIVATE_KEY_UNLOCK();
```
Comment (SparkiDev) [inline src/we_internal.c:1378]: Should have unlock and lock calls around each private key operation.

Comment (anhu) [reply src/we_internal.c:1378]: Once its unlocked, why would we need to lock and unlock it?

Comment (SparkiDev) [reply src/we_internal.c:1378]: Don't unlock it globally.
Only unlock it for the shortest amount of time.

**Comment by anhu** [comment]
Closing in favour of https://github.com/wolfSSL/wolfEngine/pull/209 .  That PR is based on feedback from @SparkiDev .

**Pull request closed without merging**